UnitTestFrameworkPkg/GoogleTest.md

```diff
@@ -872,6 +920,57 @@ extern "C" {
 }
 ```
 
+#### FunctionMockLib Mocks - Definition for non-Libraries
+The definitions of mock functions for non-libraries, or interfaces that don't have an
+inf associated with them (for example, protocols or PPIs) can be in the same header file
+as the declarations.
+
+Below the struct of mock function declarations, you can use `MOCK_INTERFACE_DEFINITION`
+and `MOCK_FUNCTION_DEFINITION`.
+For example, when creating the definition of the mock for `RngProtocol`,

```

Comment:
`RngProtocol` is not in edk2.  Change to 

mock for the function pointers in the `EFI_RNG_PROTOCOL` data structure.